doc/node-types-html/index.html.in

```diff
@@ -19,6 +19,216 @@
 
```

Comment:
Since you are removing ExamplesView.js. you should remove references that point to it otherwise you will receive errors.